server/models/menu.js

```diff
@@ -0,0 +1,63 @@
+let knex = require('../database/knex.js');
+
+
+
+
+exports.getMenu = async (req, res) => {
```

Comment:
This function name should be **getAllMenuImages** or something *more indicates* what the function returns. because you only fetch **image_url** from all of the food.

> And with every function ❤️

Reply:
ok